docs/content.zh/docs/dev/datastream/fault-tolerance/checkpointing.md

```diff
@@ -71,7 +74,13 @@ Checkpoint 其他的属性包括：
     
   - *externalized checkpoints*: 你可以配置周期存储 checkpoint 到外部系统中。Externalized checkpoints 将他们的元数据写到持久化存储上并且在 job 失败的时候*不会*被自动删除。
     这种方式下，如果你的 job 失败，你将会有一个现有的 checkpoint 去恢复。更多的细节请看 [保留 checkpoints 的部署文档]({{< ref "docs/ops/state/checkpoints" >}}#保留-checkpoint)。
-    
+
+  - *非对齐 checkpoints*: 你可以启用[非对齐 checkpoints]({{< ref "docs/ops/state/checkpointing_under_backpressure" >}}#非对齐-checkpoints)
+     以在背压时大大减少创建checkpoint的时间。这仅适用于精确一次（exactly-once）checkpoints 并且只有一个并发检查点。
+
+  - *部分任务结束的 checkpoints*： 默认情况下，即使DAG的部分已经处理完其所有记录，Flink也会继续执行 checkpoints。请参阅[重要注意事项]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}#任务结束前等待最后一次-checkpoint)以了解详细信息。
```

Comment:
```suggestion
  - *部分任务结束的 checkpoints*： 默认情况下，即使DAG的部分已经处理完它们的所有记录，Flink也会继续执行 checkpoints。请参阅[重要注意事项]({{< ref "docs/dev/datastream/fault-tolerance/checkpointing" >}}#部分任务结束后的-checkpoint)以了解详细信息。
```

Reply:
`#任务结束前等待最后一次-checkpoint` - this can not be changed because it is the the link of other section: https://nightlies.apache.org/flink/flink-docs-master/zh/docs/dev/datastream/fault-tolerance/checkpointing/#%e4%bb%bb%e5%8a%a1%e7%bb%93%e6%9d%9f%e5%89%8d%e7%ad%89%e5%be%85%e6%9c%80%e5%90%8e%e4%b8%80%e6%ac%a1-checkpoint

Reply:
I mean we should redirect it to this link: https://nightlies.apache.org/flink/flink-docs-master/zh/docs/dev/datastream/fault-tolerance/checkpointing/#%e9%83%a8%e5%88%86%e4%bb%bb%e5%8a%a1%e7%bb%93%e6%9d%9f%e5%90%8e%e7%9a%84-checkpoint

You can check with English doc[1], it is redirected to `部分任务结束后的-checkpoint` link instead of `务结束前等待最后一次-checkpoint` link.

[1] https://nightlies.apache.org/flink/flink-docs-master/docs/dev/datastream/fault-tolerance/checkpointing/#enabling-and-configuring-checkpointing

Reply:
gotcha, thanks for the clarification!